smartapps/imbrianj/door-knocker.src/door-knocker.groovy

```diff
@@ -16,8 +16,7 @@ definition(
     description: "Alert if door is knocked, but not opened.",
     category: "Convenience",
     iconUrl: "https://s3.amazonaws.com/smartapp-icons/Convenience/Cat-Convenience.png",
-    iconX2Url: "https://s3.amazonaws.com/smartapp-icons/Convenience/Cat-Convenience%402x.png",
-    pausable: true
+    iconX2Url: "https://s3.amazonaws.com/smartapp-icons/Convenience/Cat-Convenience%402x.png"
```

Comment:
any reason we took out pausable?

Reply:
Oh yeah I can add that back in now. Back in March there was a bug where pausable apps were failing to compile so I pulled it since we wanted this out right away (lol), but we've fixed that so I'll update